dc

```diff
@@ -10,6 +10,24 @@ if [ -f .env ]; then
   IFS=$'\n'
   for x in $(grep -v '^#.*' .env); do export $x; done
   IFS=$OLD_IFS
+else
+  # ensure a .env is present
+  touch .env
+fi
+
+if [ -z $(cat .env | grep -i '^SERVER_UID=') ];
+then
+  # add a SERVER_UID to .env if none was found
+  SERVER_UID=$(openssl rand -hex 8)
+  printf "# randomly generated 16 character string that must be unique across servers\nSERVER_UID=$SERVER_UID" >> .env
```

Comment:
```suggestion
  printf "\n# randomly generated 16 character string that must be unique across servers\nSERVER_UID=$SERVER_UID" >> .env
```

otherwise it appends without new line